ext/bg/js/settings/main.js

```diff
@@ -260,6 +259,7 @@ async function onOptionsUpdated({source}) {
     const options = await getOptionsMutable(optionsContext);
 
     document.querySelector('#enable-clipboard-popups').checked = options.general.enableClipboardPopups;
+    ankiTemplatesUpdateValue();
```

Comment:
Does this not `await` on purpose?

Reply:
I believe all the awaits are omitted because they are unnecessary. The anki settings are modified independently, so there is no reason to await.